### ISSUE TITLE
millipede / taupede: error message "Assertion `alpha >= 0' failed" leads to abortion (Trac #1773)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1765]
When running Taupede (icerec trunc version) reconstruction, the above problem causes the whole job to abort. Maybe returning a warning and continuing with the next frame would be better?

A test file with 2 unproblematic events and 2 events causing abortion can be found here: /data/user/jstachurska/NuTauAna/trunk_icerec/Test002-3253+3.i3.bz2 
I am using a  monopod-based seed, from "CscdL4_Monopod_noBD_4" and setting the seed.length = max(50 * I3Units.m * p.energy / I3Units.PeV, 10*I3Units.m) for a positive-length seed, and the negative of that for a negative-length seed.

Migrated from https://code.icecube.wisc.edu/ticket/1773
```json
{
    "status": "closed", 
    "changetime": "2016-07-12T16:09:01", 
    "description": "When running Taupede (icerec trunc version) reconstruction, the above problem causes the whole job to abort. Maybe returning a warning and continuing with the next frame would be better?\n\nA test file with 2 unproblematic events and 2 events causing abortion can be found here: /data/user/jstachurska/NuTauAna/trunk_icerec/Test002-3253+3.i3.bz2 \nI am using a  monopod-based seed, from \"CscdL4_Monopod_noBD_4\" and setting the seed.length = max(50 * I3Units.m * p.energy / I3Units.PeV, 10*I3Units.m) for a positive-length seed, and the negative of that for a negative-length seed.", 
    "reporter": "jstachurska", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1468339741343204", 
    "component": "combo reconstruction", 
    "summary": "millipede / taupede: error message \"Assertion `alpha >= 0' failed\" leads to abortion", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-07-06T19:12:52", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160706T19:45:21: jstachurska commented:

> Script: /data/user/jstachurska/NuTauAna/trunk_icerec/test_taupede.py

**Comment by icecube-trac** [comment]
Trac update at 20160706T20:47:25: 
* jbraun changed status from "new" to "accepted"
* jbraun changed owner from "" to "jbraun"

**Comment by icecube-trac** [comment]
Trac update at 20160712T16:09:01: 
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 148349:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148349"
> Catch negative alpha.  Abort minimization when it happens, since this indicates problems with the LLH function (probably numerical precision limits), and return a best alpha of 0.  Closes #1765.
> ```


* jbraun changed status from "accepted" to "closed"
